src/components/organisms/DataExplorer.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { Pane } from 'evergreen-ui';
+import DataResultItem from '../molecules/DataResultItem';
+import { useQuery } from '@apollo/client';
+import { METADATA, MetadataEntry, Metadata } from '../../queries/metadata';
+import Loading from '../atoms/Loading';
+import ErrorMessage from '../atoms/ErrorMessage';
+
+const DataExplorer: React.FC = () => {
+    const { data, loading, error } = useQuery<Metadata>(METADATA);
+
+    if (loading) {
+        return (
+            <Pane height="100vh">
+                <Loading size={50} />
+            </Pane>
+        );
+    }
+
+    if (error) {
+        return (
+            <Pane height="100vh">
+                <ErrorMessage message="Det oppsto en feil under henting av datasett" moreInfo={error.message} />
+            </Pane>
+        );
+    }
+
+    return (
+        <Pane>
+            {data &&
```

Comment:
If data is undefined will the page just be blank? Maybe there could be a message (ish like the Error message) saying something like "Ingen data funnet🤒" or something.

Reply:
This is handled if you opt for using the `DataWrapper`, ref.

```tsx
    if (!data) {
        return <ErrorMessage message="Ingen data mottatt fra serveren." />;
    }
```

Reply:
I dont think this should be an error. Is more feedback to the user that nothing was found for the given search input or filters once we implement these features.